.github/workflows/performance.yml

```
@@ -0,0 +1,447 @@
+  # パフォーマンス回帰検知
+  regression-detection:
+    name: Performance Regression Detection
+    runs-on: ubuntu-latest
+    needs: [baseline-performance, comprehensive-benchmark]
+    if: github.event_name == 'pull_request'
+    
+    permissions:
+      pull-requests: write
+      
+    steps:
+    - name: Download baseline results
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)